00-HTML-CSS-basics/CSS/Basics/css/style.css

```diff
@@ -0,0 +1,52 @@
+
+/*font styles*/
+/*----------*/
+
+@font-face{
+  font-family: 'MyFont';
+  src:url('../fonts/amatic/Amatic-Bold.ttf');
+  font-weight: bold;
+  font-style: normal;
+  font-size: 14px;
+ 
+}
+
+/*element selectors*/
+/*----------------*/
+html{
```

Comment:
**Minor:** As a css styling we should leave 1 white space between the selector name and the curly brace after it.